resources/views/dropdown.blade.php

```diff
@@ -2,6 +2,7 @@
     'dropdownProperty'       => 'dropdownOpen',
     'dropdownContentClasses' => 'bg-white rounded-xl shadow-lg dark:bg-theme-secondary-800 dark:text-theme-secondary-200',
     'buttonClassExpanded'    => 'text-theme-primary-500',
+    'buttonClassContracted'  => '',
```

Comment:
It's odd to indicate a dropdown being close by using "contract", as you never hear people talk about "contracting a dropdown".

```suggestion
    'buttonClassClosed'  => '',
```